### PR TITLE
FOUR-9930: Advanced Window Pane - Pane structure

### PR DESCRIPTION
**Pull request by danloa**
Implements [https://processmaker.atlassian.net/browse/FOUR-9930](https://processmaker.atlassian.net/browse/FOUR-9930)

@ryancooley  the only failing item is SonarQube that is not considering e2e tests when it evaluates the code coverage.

### CODE REVIEW

File: src/components/inspectors/NoPreviewAvailable.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <img :src="noPreviewImg" alt="no preview">
+    <h1>No Screen Asigned</h1>
+    <h2>This is a placeholder screen. Please, assign a screen to this task.</h2>
+  </div>
+</template>
+
+<script>
+
+export default {
+  props: [],
```
Comment (rodriquelca) [inline src/components/inspectors/NoPreviewAvailable.vue:12]: remove empty prop

File: src/components/inspectors/NoPreviewAvailable.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  data() {
+    return {
+      noPreviewImg: require('@/assets/noPreviewImage.png'),
+    };
+  },
+  methods: {
```
Comment (rodriquelca) [inline src/components/inspectors/NoPreviewAvailable.vue:18]: remove empty methods

File: src/components/inspectors/PreviewPanel.vue [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            v-model="selectedPreview"
+          >
+            <template #button-content>
+              <i class="fas fa-sort-down" />
+            </template>
+
+            <b-dropdown-item key="1" class="ellipsis-dropdown-item mx-auto" @click="onSelectedPreview(1)">
+              <div class="ellipsis-dropdown-content">
+                <b class="pr-1 fa-fw fas fa-file-alt" />
+                <span>Document</span>
```
Comment (rodriquelca) [inline src/components/inspectors/PreviewPanel.vue:33]: translation was missed

File: src/components/inspectors/PreviewPanel.vue [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+              </div>
+            </b-dropdown-item>
+
+            <b-dropdown-item key="2" class="ellipsis-dropdown-item mx-auto" @click="onSelectedPreview(2)">
+              <div class="ellipsis-dropdown-content">
+                <b class="pr-1 fa-fw">{ }</b>
+                <span>Object</span>
```
Comment (rodriquelca) [inline src/components/inspectors/PreviewPanel.vue:40]: translations was missed

File: src/components/inspectors/PreviewPanel.vue [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+              </div>
+            </b-dropdown-item>
+          </b-dropdown>
+          <span>Preview - {{ taskTitle }}</span>
```
Comment (rodriquelca) [inline src/components/inspectors/PreviewPanel.vue:44]: add translations

**Review by devmiguelangel — APPROVED** [review]

**Review by rodriquelca — APPROVED** [review]

**Pull request merged**